sites/svelte.dev/content/blog/2023-03-01-whats-new-in-svelte-march-2023.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "What's new in Svelte: March 2023"
+description: "SvelteHack, post-1.0 SvelteKit improvements and a huge showcase"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+March is coming in hot with a number of essential features having been added to SvelteKit following its 1.0 launch. Check out [last week's blog post](https://svelte.dev/blog/streaming-snapshots-sveltekit) for more details.
+
+We're also excited to have launched [SvelteHack](https://hack.sveltesociety.dev/) last month! Calling all Svelte developers, new and old, to make something awesome before the April 17th deadline.
+
+Let's jump into all the details...
+
+## What's new in SvelteKit
+- `$app/paths` can now be used without an app - making things like component testing with Vitest, using Storybook, etc. easier (**1.4.0**, [#8838](https://github.com/sveltejs/kit/pull/8838))
+- Adapters can now be configured at the route level (**1.5.0**, [Docs](https://kit.svelte.dev/docs/page-options#config), [#8740](https://github.com/sveltejs/kit/pull/8740))
+- The new snapshot mechanism preserves ephemeral DOM state even after navigation and page reloads (**1.5.0**,[Docs](https://kit.svelte.dev/docs/snapshots), [#8710](https://github.com/sveltejs/kit/pull/8710))
+- `OPTIONS` are now available within the server method (**1.6.0**, [Docs](https://kit.svelte.dev/docs/routing#server), [#8731](https://github.com/sveltejs/kit/pull/8731))
+- Richer error messages have been added for invalid exports (**1.7.0**, [#9055](https://github.com/sveltejs/kit/pull/9055))
+- Streaming promises are now available for server load functions ([Docs](https://kit.svelte.dev/docs/load#streaming-with-promises), [#8901](https://github.com/sveltejs/kit/pull/8901))
+
+
+## What's new in Svelte and Language Tools
```

Comment:
I noticed all the Svelte changes are unreleased. Are hoping on releasing that? If so, anything important I should call out?

Reply:
I believe @Conduitry was planning to cut a release today, so yeah we should include some stuff from it

Reply:
Cool! Will update tomorrow since @kevmodrome requested we merge on the 2nd instead 👍

Reply:
@benmccann I haven't seen a release of Svelte core today. I vote we merge as-is and we can mention all the cool features from the svelte changelog next month :)